.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
```

Comment:
Since you're updating this you might as well start adopting the best practice of using commit hashes instead of versions:

`actions/upload-artifact@65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08 #v4.6.0`

Reply:
Done